### PR TITLE
Queue manager

### PR DESCRIPTION
**Pull request by rshin8763**

### CODE REVIEW

File: config.template.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 // copy this to your config file
-module.exports = {
```
Comment (tymc7) [inline config.template.js]: Why is this removed?

**Review by tymc7 — COMMENTED** [review]
Only that one request, everything else looks good

File: config.template.js (hunk 1 above)
Comment (rshin8763) [reply config.template.js]: by mistake. I'll fix it in my next pull request.

Comment (tymc7) [reply config.template.js]: Can you fix and push the branch again? If we merge it will break the master branch

**Review by tymc7 — APPROVED** [review]
Looks good to me
